src/objects/conf/base.py

```diff
@@ -72,13 +72,24 @@
 
 # settings for sending notifications
 NOTIFICATIONS_KANAAL = "objecten"
-NOTIFICATIONS_DISABLED = config("NOTIFICATIONS_DISABLED", False)
 
 # TODO should this be moved to open-api-framework?
 # Add (by default) 5 (soft), 15 (hard) minute timeouts to all Celery tasks.
-CELERY_TASK_TIME_LIMIT = config("CELERY_TASK_HARD_TIME_LIMIT", default=15 * 60)  # hard
+CELERY_TASK_TIME_LIMIT = config(
+    "CELERY_TASK_HARD_TIME_LIMIT",
+    default=15 * 60,
+    help_text=(
+        "Task hard time limit in seconds. The worker processing the task will be "
+        "killed and replaced with a new one when this is exceeded."
+    ),
+    group="Celery",
+)  # hard
+# TODO this doesn't seem to have any effect, because we do not catch the exception anywhere?
+# https://docs.celeryq.dev/en/latest/userguide/configuration.html#task-soft-time-limit
 CELERY_TASK_SOFT_TIME_LIMIT = config(
```

Comment:
@annashamray do you know more about this? In the Celery docs, it is mentioned that after this soft time limit a `SoftTimeLimitExceeded` is raised, and @Viicos mentioned that this time limit is needed for `notifications_api_common`, but I don't see that exception being caught in that library

Reply:
I've found the related PR (https://github.com/maykinmedia/objects-api/pull/362) but it's not much of help.
My guest that it is a copy-paste from Open Forms and I also haven't found any catching of `SoftTimeLimitExceeded`

Reply:
In that case I'll remove this envvar entirely